README.md

```diff
@@ -1,15 +1,24 @@
 # Package Template
 
-[![Greenkeeper badge](https://badges.greenkeeper.io/5app/package-template.svg)](https://greenkeeper.io/)
+[![Known Vulnerabilities](https://snyk.io/test/github/5app/package-template/badge.svg)](https://snyk.io/test/github/5app/package-template)
 [![CircleCI](https://circleci.com/gh/5app/package-template.svg?style=shield)](https://circleci.com/gh/5app/package-template)
 
 This is a template for starting new JS projects at 5app.
 
-## Getting started
+## What now
 
-Run `npm init` to configure package.json
+1. Run `npm init` to configure [./package.json](./package.json]).
```

Comment:
Wow, I didn't know using `1.` in all list items will be replaced with the right numbers automatically. Thanks 🙏